src/page-not-found/PageNotFound.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styled from "styled-components";
+import { HeadingTitle, HeadingDescription } from "../heading";
+
+const Wrapper = styled.div`
+  padding: 88px 0;
+  text-align: center;
+`;
+
+export default function PageNotFound() {
+  return (
+    <Wrapper>
+      <HeadingTitle>Error 404: Page not found.</HeadingTitle>
+      <HeadingDescription>
+        <p>The page that you are looking for is missing.</p>
+        <p>
+          Go to <a href="https://docr.nd.gov">docr.nd.gov</a>
```

Comment:
Do we really want to redirect all the way back to the DOCR and not the root of the app?

Reply:
I will defer to @hobuobi on this one, I wasn't really part of the content conversations for these messages

Reply:
Sorry about the late response here! 
We should definitely refer to the root of the app in addition to this (probably as the first option). I'd suggest the following: 

> Click [here](http://dashboard.docr.nd.gov) to return to the homepage of the dashboard, or visit the North Dakota Department of Corrections at [docr.nd.gov](http://docr.nd.gov).